demosplan/DemosPlanCoreBundle/Controller/Segment/SegmentsExportController.php

```diff
@@ -74,11 +73,7 @@ static function () use ($procedure, $statement, $exporter, $tableHeaders) {
             }
         );
 
-        $filename = $slugify->slugify($procedure->getName())
-            .'-'
-            .$statement->getExternId().'.docx';
-
-        $this->setResponseHeaders($response, $filename);
```

Comment:
'.docx' is missing. I don't get a docx file when I export detail view.

Reply:
done 2d4964e27d8185448811b46108ec07f8f538a725